### ISSUE TITLE
Zimit2: add support for web/service workers

### ISSUE THREAD

**Issue by benoit74** [openzim/warc2zim#231]
Currently, warc2zim2 does not include `wombatWorker.js` while it said on https://github.com/webrecorder/wombat that this file _"is not a bundle per say but rather a flat file that applies the minimal set of overrides necessary to ensure that JavaScript web and service worker's operate as expected in both non-proxy recording and replay."_

We must setup a test case on test website (both cases probably, or at least a web worker since we know service workers require HTTPS and might hence be difficult to make work in a ZIM), check it is not working (I do not see why it would work if wombat has a dedicated component for it) and does the necessary changes (at least include wombatWorker.js in `openzim.toml` assets to download, but I'm pretty sure it will not be sufficient and we will have other stuff to fix as well.

**Comment by Jaifroid** [comment]
I agree we need to add `wombatWorker.js`. This would primarily be for Web Workers which are increasingly necessary to replay many Web sites accurately.

Of course Service Workers can't be emulated, and won't work in all the contexts where Zimit 1 (i.e. the Replay Service Worker) can't run. The saving grace here is that almost all Web sites treat Service Worker support as optional, and degrade gracefully.

**Comment by benoit74** [comment]
See https://github.com/openzim/zim-requests/issues/1086 for an example of a website with (what looks like) a service worker and does not work at all inside the ZIM
